back-end/API/README.md

```diff
@@ -38,6 +38,21 @@ CREATE TABLE IF NOT EXISTS job_offer (
     employerId INT NOT NULL,
     scheduleId INT NOT NULL
 );
+CREATE TABLE Employers (
```

Comment:
la section "Setting up the database" est-elle vraiment utile? La migration le fait déjà.

Reply:
C'est fait